app/assets/javascripts/AngularConfig.js.coffee

```diff
@@ -133,6 +134,7 @@ angular.module('dahlia.controllers',['ngSanitize', 'angular-carousel', 'ngFileUp
       url: '/sign-in?expiredUnconfirmed&expiredConfirmed'
       params:
         newAccount: {squash: true}
+        skipConfirm: { squash: true, value: false }
```

Comment:
@chrisdolendo squash is for params that you don't actually send via the browser URL but are just sent programmatically via `$state.go`